app/src/main/java/io/lbry/browser/adapter/ClaimListAdapter.java

```diff
@@ -430,7 +430,6 @@ public void onClick(View view) {
         Helper.setViewVisibility(vh.loadingTextPlaceholder2, item.isLoadingPlaceholder() ? View.VISIBLE : View.GONE);
         Helper.setViewVisibility(vh.titleView, !item.isLoadingPlaceholder() ? View.VISIBLE : View.GONE);
         Helper.setViewVisibility(vh.publisherView, !item.isLoadingPlaceholder() ? View.VISIBLE : View.GONE);
-        Helper.setViewVisibility(vh.publishTimeView, !item.isLoadingPlaceholder() ? View.VISIBLE : View.GONE);
```

Comment:
This line is actually required for cases where the view type is a placeholder view. I've added an additional check for the pending flag to prevent the conditions from overlapping.